### PR TITLE
code_of_conduct:Add summary of contributor code of conduct

### PR DESCRIPTION
**Pull request by Ginohmk**
Fixes:##1346

PR Checklist:
- [x] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [x] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [x] Add a summary of the code of conduct to the readme
- [x] Add copy functionality to the steps of command to setup dev environment.

### CODE REVIEW

**Comment by Ginohmk** [comment]
@beckermr @kathatherine . Please review my PR, so changes can be made if required. Thank you

**Comment by Ginohmk** [comment]
@BastianZim

File: src/README.md [hunk 1]
```diff
@@ -1,15 +1,95 @@
...
+
+```
+cd ../src
+```
+
+```
+make html
+```
+
+# Contributor NumFocus Code of Conduct
```
Comment (kathatherine) [inline src/README.md:28]: I don't think we need NumFocus's Code of Conduct in here. conda-forge would need to have its own. We don't want to just copy theirs.

**Review by kathatherine — CHANGES_REQUESTED** [review]

**Comment by Ginohmk** [comment]
@kathatherine @BastianZim @beckermr Changes requested have been made.

File: src/README.md (hunk 1 above)
Comment (kathatherine) [reply src/README.md]: Ooh, looks like you already did that. Cool.

Comment (Ginohmk) [reply src/README.md]: Thank you for your feedback @kathatherine
 if they are any other thing I need to add to the readme, please let me know, so I wrap this PR up and work on another issue.

 I really want to constantly contribute to this project.

File: src/README.md [hunk 2]
```diff
@@ -1,15 +1,95 @@
...
+- Trolling or insulting and derogatory comments.
+- Public or private harassment.
+- Sharing private content, such as emails sent privately or non-publicly, or direct message history, without the sender’s consent.
+- Posting sexually explicit or violent material.
+- Other unethical or unprofessional conduct
+- Advocating for, or encouraging, any of the above behaviors
+
+## Reporting a Misconduct
+
+- You can reach the Project maintainers.
```
Comment (kathatherine) [inline src/README.md:62]: If you're including this section, you'll need to put a way to reach the conda-forge maintainers. You can link to the https://conda-forge.org/docs/orga/getting-in-touch.html page and then people can decide how they want to communicate an issue from there.

File: src/README.md [hunk 3]
```diff
@@ -1,15 +1,95 @@
...
+Examples of unacceptable behavior by participants include:
+
+- The use of sexualized language or imagery.
+- Violent or intimidating threats or language directed against another person.
+- Trolling or insulting and derogatory comments.
+- Public or private harassment.
+- Sharing private content, such as emails sent privately or non-publicly, or direct message history, without the sender’s consent.
+- Posting sexually explicit or violent material.
+- Other unethical or unprofessional conduct
+- Advocating for, or encouraging, any of the above behaviors
```
Comment (kathatherine) [inline src/README.md:58]: Lines 57 and 58 above need periods.

File: src/README.md [hunk 4]
```diff
@@ -1,15 +1,95 @@
...
+
+## Reporting a Misconduct
+
+- You can reach the Project maintainers.
+
+## Enforcement: What Happens After a Report is Filed
+
+- Reviewing the Report.
+- Contacting the Person Reported.
```
Comment (kathatherine) [inline src/README.md:67]: "report", "person", and "reported" in the lines above don't need to be capitalized.

File: src/README.md [hunk 5]
```diff
@@ -1,15 +1,95 @@
...
+
+## Response and Potential Consequences
+
+- Nothing (if we determine that no violation occurred).
+- Private feedback or reprimand from Conda-forge to the individual(s) involved.
+- Removing a person from Conda-forge membership or other formal affiliation.
+- Publishing an account of the harassment and calling for the resignation of the alleged harasser from their responsibilities (usually pursued by people without formal authority: may be called for if the person is the event leader, or refuses to stand aside from the conflict of interest, or similar)
+- Any other response that Conda-forge deems necessary and appropriate to the situation
+
```
Comment (kathatherine) [inline src/README.md:76]: Lines 74 and 75 need periods for consistency.

File: src/README.md [hunk 6]
```diff
@@ -1,15 +1,95 @@
...
+- _[admin-requests](https://github.com/conda-forge/admin-requests)_
+- _[conda-forge-repodata-patches-feedstock](https://github.com/conda-forge/conda-forge-repodata-patches-feedstock)_
+- _[conda-forge-pinning-feedstock](https://github.com/conda-forge/conda-forge-pinning-feedstock)_
+- _[docker-images](https://github.com/conda-forge/docker-images)_
+- _[conda-forge-ci-setup-feedstock](https://github.com/conda-forge/conda-forge-ci-setup-feedstock)_
+- _[marketing](https://github.com/conda-forge/marketing)_
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Conda-forge][code of conduct]
```
Comment (kathatherine) [inline src/README.md:93]: Are you meaning for the [homepage] link below to be a part of the sentence on line 93 above?

**Review by kathatherine — CHANGES_REQUESTED** [review]

File: src/README.md (hunk 6 above)
Comment (Ginohmk) [reply src/README.md:93]: Yes, it is intended to, I just modified it to do exactly that now (linking to [link](https://conda-forge.org/docs/orga/governance.html)), on my next commit

Comment (Ginohmk) [reply src/README.md:93]: it is an attribution. The link takes the viewer to the conda forge guidance page which has a link to NumFocus.

**Comment by Ginohmk** [comment]
@kathatherine requested changes have been made.

File: src/README.md [hunk 7]
```diff
@@ -1,15 +1,97 @@
...
+- The use of sexualized language or imagery.
+- Violent or intimidating threats or language directed against another person.
+- Trolling or insulting and derogatory comments.
+- Public or private harassment.
+- Sharing private content, such as emails sent privately or non-publicly, or direct message history, without the sender’s consent.
+- Posting sexually explicit or violent material.
+- Other unethical or unprofessional conduct.
+- Advocating for, or encouraging, any of the above behaviors.
+
+## Reporting a Misconduct
```
Comment (kathatherine) [inline src/README.md:62]: I'd take "a" out here.

File: src/README.md [hunk 8]
```diff
@@ -1,15 +1,97 @@
...
+
+- You can reach the Project [maintainers](https://conda-forge.org/docs/orga/getting-in-touch.html).
```
Comment (kathatherine) [inline src/README.md:64]: How about, "You can reach the project maintainers [here]. And have "here" be the link. I think that will read better.

File: src/README.md (hunk 6 above)
Comment (kathatherine) [reply src/README.md:95]: Ah, I understand now. The square brackets around "code of conduct" should be parenthesis. That should cause it to link correctly.

**Review by kathatherine — CHANGES_REQUESTED** [review]
A few more things and you should be good!

File: src/README.md (hunk 6 above)
Comment (Ginohmk) [reply src/README.md:95]: Yes, the way I did it is a more fancy way of creating a markdown link :smile:
 But I think your idea is better, so as to maintain consistency in the markdown.

File: src/README.md (hunk 8 above)
Comment (Ginohmk) [reply src/README.md:64]: Yes i agree with you

**Comment by Ginohmk** [comment]
@kathatherine Changes have been made.

**Review by kathatherine — APPROVED** [review]
Looks good! Thanks

**Comment by Ginohmk** [comment]
Thank you @kathatherine.

**Comment by jaimergp** [comment]
Superseded by https://github.com/conda-forge/conda-forge.github.io/pull/1640

**Pull request closed without merging**